.pre-commit-config.yaml

```diff
@@ -1,8 +1,29 @@
+fail_fast: true
+minimum_pre_commit_version: "2"
```

Comment:
Do we also want to include
```yml
default_language_version:
    python: python3.9
```
?

Reply:
i don't think so, because its just running make commands. We already have that information in the `setup.py` file